oclubs/static-dev/js/oclubs.js

```diff
@@ -144,16 +170,97 @@
 				} );
 
 			$( '#emptyclose, #closebtn' )
-				.click( function() {
+				.click( function () {
 					$( '#sidenav, #emptyclose' )
 						.css( 'width', '0' );
 					$( '#floatmenu' )
 						.fadeIn();
 				} );
 
 			$( '.large_container select.mobileselect' )
-				.change( function() {
+				.change( function () {
 					window.location = this.value;
 				} );
+
+			function showScrollToTopBtn() {
+				if ( document.body.scrollTop > 400 || document.documentElement.scrollTop > 400 ) {
+					document.getElementById( 'scroll_to_top_btn' )
+						.style.display = 'block';
+				} else {
+					document.getElementById( 'scroll_to_top_btn' )
+						.style.display = 'none';
+				}
+			}
+
+			window.onscroll = function () {
+				showScrollToTopBtn();
```

Comment:
Is this the only use of `showScrollToTopBtn`. If so, could you inline the function?